src/Homepage/Button/SearchButton.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import './_search-button.scss';
+
+import { Link } from 'react-router-dom';
```

Comment:
This line should be placed after the first line. And please try not to use more than 1 empty lines.